keyboards/stront/40keys/config.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* 40keys overrides */
+#undef LCD_DC_PIN
+#define LCD_DC_PIN GP27
+#undef LCD_RST_PIN
+#define LCD_RST_PIN GP12
+#undef BACKLIGHT_PWM_DRIVER
+#define BACKLIGHT_PWM_DRIVER PWMD6
+
+/* Bootmagic on top outer pinky key */
+#define BOOTMAGIC_ROW 3
+#define BOOTMAGIC_COLUMN 0
```

Comment:
Defaulting to top left is more consistent. Adding in a "right" version is also an option. If this is required, it should be configured in keyboard.json
```suggestion
```

Reply:
40keys version had to be wired differently, so [3, 0] is the top left key. I added that so that it matches the 38keys version.
Moved to keyboard.json, and added "right" version.